pyproject.toml

```diff
@@ -278,6 +279,7 @@ ignore = [
     "ANN401", # any-type
     "PT011", # pytest-raises-too-broad
 ]
+preview = true
```

Comment:
Why do we want to enable unstable features?

Reply:
It's required for many of the FURB rules.